### PR TITLE
Feature/update rpi imager doc

### PR DESCRIPTION
**Pull request by keesdaamen**
Solves issue #247. Questions / discussions labeled with REVIEW.

### CODE REVIEW

File: docs/user/startup.md [hunk 1]
```diff
@@ -37,72 +37,31 @@ When connecting the BrewPi Spark
...
+![RPi-imager](../images/rpi-imager.png)
 
-![Imager](../images/imager.png)
+Select Raspberry Pi OS Lite (32-bit), select your SD card, and hit **'Ctrl-Shift-X'** for the advanced menu. The advanced menu is available in the Raspberry Pi Imager v1.6 and higher.
 
-For more information, see the [official Raspberry install guide](https://www.raspberrypi.org/documentation/installation/installing-images/README.md).
+![RPi-imager-adv](../images/rpi-imager-adv.png)
 
-## Step 2: Enable SSH and WiFi
+REVIEW: plaatje van het advanced menu van de RPi website. Zelf een plaatje maken kan, maar is veel werk i.v.m. scrollbar en geen 'automatic scrollbar screen capture' gevonden op linux.
```
Comment (steersbob) [inline docs/user/startup.md]: Discussion notes are not content. It's best to add them as comment in the Github PR (like this).

On-topic: an image with the scrollbar active is fine. The image serves as a confirmation that the user is looking at the right menu, and we don't want it to take up too much vertical space.

File: docs/user/startup.md [hunk 2]
```diff
@@ -37,72 +37,31 @@ When connecting the BrewPi Spark
...
-The `ssh` file enables SSH, just by being there.
-
-To configure WiFi, open `wpa_supplicant.conf` in a text editor. The file contents should be:
-
-```
-country=YOUR_COUNTRY_CODE
-
-ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+In the advanced menu you want to define a few settings:
+- **Set hostname:** set a hostname to easily connect to your Pi on your network. Clear and short names work best. Default hostname is `raspberrypi`.
```
Comment (steersbob) [inline docs/user/startup.md]: Missing article: **The** default hostname...

File: docs/user/startup.md [hunk 3]
```diff
@@ -37,72 +37,31 @@ When connecting the BrewPi Spark
...
+- **Enable SSH:** you want to enable SSH to remotely connect to your Pi through a terminal. Select *Use password authentification* and set your password.
```
Comment (steersbob) [inline docs/user/startup.md]: authentification -> authentication

File: docs/user/startup.md [hunk 4]
```diff
@@ -37,72 +37,31 @@ When connecting the BrewPi Spark
...
+- **Configure wifi:** here you configure the network name (SSID) & password of your wifi (if applicable). Set the Wifi country setting to your country to match the wifi channels of your access point.
```
Comment (steersbob) [inline docs/user/startup.md]: Can leave out "if applicable". There really is no excuse for exposing hardware controls on an unsecured wifi network.

The rest of the guide seems to somewhat consistently use the `WiFi` spelling.

File: docs/user/startup.md [hunk 5]
```diff
@@ -37,72 +37,31 @@ When connecting the BrewPi Spark
...
+- **Set locale settings:** set your locale settings to define the system time of your Pi and the layout of your keyboard.
```
Comment (steersbob) [inline docs/user/startup.md]: Goes for the points above too: the instructions can be somewhat shortened. In this case, you can skip "set your locale settings to...", as it just repeats the information from before the colon.

File: docs/user/startup.md [hunk 6]
```diff
@@ -37,72 +37,31 @@ When connecting the BrewPi Spark
 You can also install Brewblox on a Synology NAS, desktop computer, or laptop - as long as it's using Linux.
 :::
 
-## Step 1: Format the microSD card
+## Step 1: Prepare the microSD card
```
Comment (steersbob) [inline docs/user/startup.md]: Step 1 and 2 have become step 1. "Format" still seems the appropriate wording.

File: docs/user/startup.md [hunk 7]
```diff
@@ -121,26 +80,27 @@ If you're unfamiliar with SSH, [this tutorial](https://www.howtogeek.com/311287/
...
 
 ```bash
-ssh pi@raspberrypi
+ssh pi@custom_hostname
 ```
 
-If `raspberrypi` is not a recognized host or address, you can use the instructions below to get the IP address of your Pi.
+If the hostname of your Pi is not recognized on your network, you can use the instructions below to get the IP address of your Pi.
+
+REVIEW: betere manier om, om te gaan met het custom hostname verhaal?
```
Comment (steersbob) [inline docs/user/startup.md]: Same as before: review notes don't belong in content.

we can use `raspberrypi` in examples, if we also add a line here about users substituting the default `raspberrypi` with the hostname they chose in step 1.

File: docs/user/startup.md [hunk 8]
```diff
@@ -121,26 +80,27 @@ If you're unfamiliar with SSH, [this tutorial](https://www.howtogeek.com/311287/
...
-Often, your network already knows the address for the `raspberrypi` name.
+Often, your network already knows the address of your Pi by its hostname. If your haven't changed it at step [1](#step-1-prepare-the-microsd-card), the default hostname of your Pi is `raspberrypi`.
 
 If using the name doesn't work, there are multiple tools to discover the IP address.
 
 We like the [Fing](https://www.fing.com/products) app (available on both [iOS](https://apps.apple.com/us/app/fing-network-scanner/id430921107) and [Android](https://play.google.com/store/apps/details?id=com.overlook.android.fing&hl=en)), but you can also use your router's web interface to find the addresses of all connected devices.
 A tutorial for that can be found [here](https://helpdeskgeek.com/how-to/determine-computers-connected-to-wireless-network/).
 
-## Step 4: Install Brewblox
+## Step 3: Install Brewblox
```
Comment (steersbob) [inline docs/user/startup.md:99]: I did a quick check, and there's a reference to "step 4 (Install Brewblox)" in user/system_upgrades.md

**Review by steersbob — CHANGES_REQUESTED** [review]
Looks good. Some minor changes and typo fixes are required.

File: docs/user/startup.md (hunk 4 above)
Comment (keesdaamen) [reply docs/user/startup.md]: "if applicable" is not meant for just the password, but for using WiFi to connect to the Spark at all. I'll change it to: "Omit if you plan to use USB." Furthermore: spelling in all user guides:
- wifi 23
- Wifi 60
- WiFi 13

Of which WiFi has 2 additions by my new introductions... Which do you want to use? I intend to follow the spelling of the RPi Imager software in naming the options

File: docs/user/startup.md (hunk 8 above)
Comment (keesdaamen) [reply docs/user/startup.md:99]: Ok, thanks. It's fixed. I found a reference to now-non-existing Step 8 within the document as well. Searched the other User manuals too.

File: docs/user/startup.md (hunk 4 above)
Comment (steersbob) [reply docs/user/startup.md]: These wifi settings are for the Pi, not the Spark. SSH over USB is technically possible, but not a realistic option.

Googling "wifi spelling" yields that the official spelling is "Wi-Fi", but that the "Wifi", "WiFi", and "wifi" alternatives are all common enough to be acceptable.

Leave it for now, and I'll make an issue to make an official decision and refactor across all repositories.

**Review by steersbob — APPROVED** [review]

**Pull request merged**